src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -140,6 +140,42 @@
       (when (:show-nih-link-warning? props)
         [nih-link-warning/NihLinkWarning])])})
 
+;; GAWB-666 Globally show Queued and Cromwell (active) counts
+(defn queue-status-counts [workflow-counts]
+  (let [count-map (get workflow-counts "workflowCountsByStatus" {})
+        queue-time (get workflow-counts "estimatedQueueTimeMS" 0)]
+    {:queue-time queue-time
+     :queued (apply + (map count-map ["Queued" "Launching"]))
```

Comment:
These don't match the counts from `launch_analysis.cljs`, and there's a discrepancy from what Rawls returns as well.

The Rawls endpoint returns only its `queuedStatuses` and `runningStatuses` so `:terminal` here will always be zero.  This is misleading.

`launch_analysis.cljs` drops the `Launching` status instead of adding it to `Queued`.  I think choosing either way is acceptable, but we should be consistent by reusing `queue-status-remap` or similar.